lib/task.js

```diff
@@ -702,13 +702,13 @@ export class Task {
         await this.states.started(this);
       }
       catch (e) {
-        debug(e.stack);
         return await this.abortRun(
           'states_failed',
           this.fmtErrorLog(
             'Task was aborted because states could not be started ' +
-            `successfully. ${e.stack || e}`
-          )
+            `successfully. ${e}`
+          ),
```

Comment:
The line above I think I want to be e.message. This e is passed to abort run and will be logged to paper trail. I want the whole error passed there